apis/validator/ssv/stream.yaml

```diff
@@ -0,0 +1,86 @@
+get:
+  operationId: "ssv_stream"
+  summary: Stream of attestations to sign from SSV node
+  tags:
+    - SSV
+  description: |
+    Provides endpoint to subscribe to attestations to sign from an SSV node. Only provided by SSV nodes.
+    Two types of events: 
+      "sign_*": Attestations/blocks should be signed and sent to the appropriate beacon node API for publication
+      "check_*": Message should be checked for conflicts with slashing protection; response to be sent to approve_* endpoint
+    Consumers should use [eventsource](https://html.spec.whatwg.org/multipage/server-sent-events.html#the-eventsource-interface)
+    implementation to listen on those events.
+  parameters:
+    - name: topics
+      in: query
+      required: true
+      description: Event types to subscribe to
+      schema:
+        type: array
+        items:
+          type: string
+          uniqueItems: true
+          enum:
+            - sign_block
+            - sign_attestation
+            - check_block
+            - check_attestation
```

Comment:
@dankrad  should include sign aggregation as well

Reply:
What do you man signature aggregation? The reconstruction of the threshold signature should be done by the SSV node, the VC doesn't need to do anything about it

Reply:
if that validator has an aggregation duty for a particular epoch we should be able to sign the proof.

Reply:
Ah right, I forgot about attestation aggregation proofs. Yes, these need to be added then.